github/filecoin-project.yml

```diff
@@ -5206,10 +5206,32 @@ teams:
       # ATTN: members are expected to:
       #  - be familiar with GitHub Management
       #  - be ready to triage/review org configuration change request in github-mgmt
+      # The individuals below are listed as "maintainers" rather than "members" because they are filecoin-project owners/admins.
+      # GitHub will auto-bump their team privileges anyway if we don't manually.
       maintainer:
+        # Why @galargh?
+        # 1. He has deep knowledge of the tool and its use as the creator.
+        # 2. He is co-founder of IPDX, which has a contract in 2024 to support developer productivity in the filecoin-project ecosystem.
         - galargh
+        # Why @jennijuju?
```

Comment:
Hrm I guess I’m fine with this but also hoping there are more people that is stewarding this so im not blocking any urgent requests. TLDR maybe onboard and  get two or more people from the project owner group to this workflow

Reply:
Yes, that is the plan.  We'll get some others when holistically attacking org ownership in https://github.com/filecoin-project/github-mgmt/issues/47 .  (This group needs to be looked at in light of who is also an org owner/admin as this group can self-service elevant themselves and the org admin/owners can self-service make changes here as well.).